src/main/java/se/sundsvall/disturbance/api/model/Category.java

```diff
@@ -4,11 +4,36 @@
 
 @Schema(description = "Category model", enumAsRef = true)
 public enum Category {
-	COMMUNICATION,
-	DISTRICT_COOLING,
-	DISTRICT_HEATING,
-	ELECTRICITY,
-	ELECTRICITY_TRADE,
-	WASTE_MANAGEMENT,
-	WATER;
+
+	COMMUNICATION("COMMUNICATION"),
+	DISTRICT_COOLING("DISTRICT_COOLING"),
+	DISTRICT_HEATING("DISTRICT_HEATING"),
+	ELECTRICITY("ELECTRICITY"),
+	ELECTRICITY_TRADE("ELECTRICITY_TRADE"),
+	WASTE_MANAGEMENT("WASTE_MANAGEMENT"),
+	WATER("WATER");
+
+	private final String value;
+
+	Category(String value) {
+		this.value = value;
+	}
+
+	public String getValue() {
+		return value;
+	}
+
+	public static Category fromValue(String value) {
```

Comment:
Doesn't the provided `valueOf(String value)` (that all enums have) do exactly this?